packages/manager/modules/telecom-spare/rollup.config.js

```diff
@@ -0,0 +1,26 @@
+import rollupConfig from '@ovh-ux/component-rollup-config';
```

Comment:
rollup config can be removed

Reply:
Does it mean I need to remove rollup.config.js file ?

Reply:
Yes if you point to `./src/index.js`, webpack will build your sources so you don't need a `rollup` configuration